GramAddict/core/filter.py

```diff
@@ -59,6 +63,24 @@ def check_profile(self, device, username):
         field_specific_alphabet = self.conditions.get(FIELD_SPECIFIC_ALPHABET)
         field_min_posts = self.conditions.get(FIELD_MIN_POSTS)
 
+        follow_button_text = self._get_follow_button_text(device)
+
+        if field_skip_following:
+            if follow_button_text == "Following":
+                logger.info(
+                    f"You follow @{username}, skip.",
+                    extra={"color": f"{Fore.GREEN}"},
+                )
+                return False
+
+        if field_skip_follower:
+            if follow_button_text == "Follow Back":
```

Comment:
Please, check my suggestion in views.py. If we implement that, there is no need to hardcode strings `Following`and `Follow back` here in this file.